.tekton/pull-request.yaml

```diff
@@ -150,6 +150,20 @@ spec:
                 # Perform cleanup of resources created by gitops service
                 oc delete --ignore-not-found deployment --all -n $(params.e2e_test_namespace)
                 oc delete --ignore-not-found eventlisteners --all -n $(params.e2e_test_namespace)
+      - name: check-task-pipeline-repo-existence
+        taskSpec:
+          steps:
+            - name: fail-when-repo-is-missed
+              image: quay.io/redhat-appstudio/pull-request-builds:build-definitions-utils-{{revision}}
```

Comment:
can be replaced by registry.redhat.io/openshift4/ose-cli:v4.11 - in case that you remove yq command

Reply:
use your redhat.com credentials